src/main/webapp/js/instructorFeedbackEdit/visibilityOptions.js

```diff
@@ -231,62 +349,56 @@ function updatePreviewTab($containingForm) {
-    var $previewTab = $containingForm.find('.visibilityMessage');
+    var $visibilityMessageDiv = $containingForm.find('.visibilityMessage');
     
     if (previousFormDataMap[questionNum] === formData) {
-        $editTab.hide();
-        $previewTab.show();
         return;
     }
 
     // empty current visibility message in the form
-    $previewTab.html('');
+    $visibilityMessageDiv.html('');
     
     var url = '/page/instructorFeedbackQuestionvisibilityMessage';
     $.ajax({
         type: 'POST',
         url: url,
         data: formData,
         success: function(data) {
-            updateToggleVisibilityPreviewButton($containingForm, true);
-            
             // update stored form data
             previousFormDataMap[questionNum] = formData;
             
-            $previewTab.html(formatPreviewTabHtml(data.visibilityMessage));
-            $previewTab.show();
-            $editTab.hide();
+            $visibilityMessageDiv.html(formatVisibilityMessageDivHtml(data.visibilityMessage));
         },
         error: function() {
-            updateToggleVisibilityPreviewButton($containingForm, false);
-            $containingForm.find('.visibilityOptionsLabel').click();
+            showAjaxErrorMessage($containingForm);
         }
     });
 }
 
-function updateToggleVisibilityPreviewButton($containingForm, isLoadSuccessful) {
-    var $visibilityPreviewButton = $containingForm.find('.visibilityMessageButton');
-    
-    var $radioInput = $visibilityPreviewButton.find('input[type="radio"]');
-    var icon = '<span class="glyphicon glyphicon-'
-               + (isLoadSuccessful ? 'eye-open' : 'warning-sign')
-               + '"></span>';
-    var message = isLoadSuccessful ? 'Preview Visibility'
-                                   : 'Visibility preview failed to load. Click here to retry.';
-    
-    $visibilityPreviewButton.html(icon + ' ' + message)
-                            .prepend($radioInput);
-}
-
-function formatPreviewTabHtml(visibilityMessage) {
-    var htmlString = 'This is the visibility as seen by the feedback giver.';
-    htmlString += '<ul class="background-color-warning">';
+function formatVisibilityMessageDivHtml(visibilityMessage) {
+    var htmlString = 'This is the visibility hint as seen by the feedback giver:';
+    htmlString += '<ul class="text-muted background-color-warning">';
     for (var i = 0; i < visibilityMessage.length; i++) {
         htmlString += '<li>' + visibilityMessage[i] + '</li>';
     }
     htmlString += '</ul>';
     return htmlString;
 }
 
+/**
+ * Update visibility message div with error message and add onclick event for re-loading the visibility message
+ */
+function showAjaxErrorMessage($containingForm) {
+    var $visibilityMessageDiv = $containingForm.find('.visibilityMessage');
+
+    var htmlString = 'This is the visibility hint as seen by the feedback giver:';
+    htmlString += '<ul class="text-muted background-color-warning">';
```

Comment:
Perhaps make this an `<a>` instead, as users might be more inclined to try clicking on that rather than a `<li>`